wgsl/index.bs

```diff
@@ -6668,6 +6668,109 @@ When an identifier is used as a [=syntax/callable=] item, it is one of:
     | [=syntax/bitwise_expression=]
 </div>
 
+## Operator Precedence and Associativity ## {#operator-precedence-associativity}
+
+This entire subsection is non-normative.
+
+Operator precedence and associativity in right-hand side WGSL expressions
+emerge from their grammar in summary. Right-hand expressions group operators to organize them,
+as illustrated by the following diagram:
+
+<div class=operators1>
+  <figure>
+    <figcaption>Operator precedence and associativity graph</figcaption>
+    <object type="image/svg+xml" data="img/operators_1.mmd.svg"></object>
+  </figure>
+</div>
+
+To promote readability through verbosity, the following groups do not associate with other groups:
+
+* Short-circuit OR (can associate with self and relational weakly),
+* Short-circuit AND (can associate with self and relational weakly),
+* Binary OR (can associate with self and unary weakly),
+* Binary AND (can associate with self and unary weakly),
+* Binary XOR (can associate with self and unary weakly).
+
+And the following groups do not associate with themselves:
+
+* Shift (can associate with unary weakly),
+* Relational (can associate with additive and shift weakly).
+
+Associating both group sections above requires parentheses to set the relationship explicitly.
+The following exemplifies where these rules render expressions invalid in comments:
+
+<div class='example wgsl function-scope' heading='Operator precedence corner cases'>
+  <xmp>
+    let a = x & (y ^ (z | w)); // Invalid: x & y ^ z | w
+    let b = (x + y) << (z >= w); // Invalid: x + y << z >= w
+    let c = x < (y > z); // Invalid: x < y > z
+    let d = x && (y || z); // Invalid: x && y || z
+  </xmp>
+</div>
+
+The following table summarizes operator precedence, associativity, and binding,
```

Comment:
I think binding could use an explanation, especially what the values in the table mean.

Reply:
I'd actually also appreciate alternative naming suggestions as this is practically something we introduce to be able to linearize the operators. I tried to add some minor explanation. Thanks!